### PR TITLE
Add support send stats to Datadog datasource

### PR DESCRIPTION
**Pull request by niels-s**
Implements another datasource to export the Kafka stats to.

### CODE REVIEW

File: cmd/consumer_offsets/main.go [hunk 1]
```diff
@@ -110,6 +147,97 @@ func main() {
...
+				if gOffset >= 0 {
+					totalGroupOffset += gOffset
+					totalLag += lag
+				}
+
+				pTags := make([]string, len(tags))
+				copy(pTags, tags)
+				pTags = append(pTags, "partition:"+strconv.Itoa(int(partitionOffset.Partition)))
+
+				err := client.Gauge("kafka.consumer_group.offsets", gOffset, pTags, 1)
```
Comment (jurriaan) [inline cmd/consumer_offsets/main.go]: `offsets` -> `offset`?

File: cmd/consumer_offsets/main.go [hunk 2]
```diff
@@ -69,6 +71,31 @@ func getInfluxClient(urlStr string) (client influxdb.Client, batchConfig influxd
...
+
+	if u.Scheme != "dogstatsd" {
+		log.Fatalf("error: we expect dogstatsd url to start with 'dogstatsd' scheme but got %v", u.Scheme)
+	}
+
+	if len(u.Path) == 0 {
+		log.Fatalln("We expect dogstatsd to have a path to indicate the cluster name")
+	}
+
+	clusterName := u.Path[1:]
```
Comment (jewilmeer) [inline cmd/consumer_offsets/main.go]: clustername? can you explain what that does. Isn't it a namespace?

File: cmd/consumer_offsets/main.go [hunk 3]
```diff
@@ -93,6 +120,16 @@ func main() {
 			groupOffsets, topicOffsets := kafkatools.FetchOffsets(client, sarama.OffsetNewest)
 			writeToInflux(influxClient, batchConfig, groupOffsets, topicOffsets)
 		}
+	} else if docOpts["--dogstatsd"] != nil {
+		dogstatsdClient, clusterName := getDogStatsdClient(docOpts["--dogstatsd"].(string))
+
+		ticker := time.NewTicker(time.Second)
+		for range ticker.C {
+			log.Println("Sending metrics to InfluxDB")
```
Comment (jewilmeer) [inline cmd/consumer_offsets/main.go]: You mean datadog? :-)

**Comment by niels-s** [comment]
@jurriaan @jewilmeer addressed the comments

**Review by jewilmeer — COMMENTED** [review]
LGTM

**Review by jewilmeer — APPROVED** [review]

**Review by koenbollen — APPROVED** [review]
Looks good to me. 👍

**Comment by niels-s** [comment]
fyi I didn't come up with this myself but borrowed a lot from https://github.com/packetloop/datadog-agent-burrow/blob/develop/checks.d/burrow.py

**Pull request merged**